packages/gatsby/src/schema/infer/example-value.js

```diff
@@ -100,7 +100,12 @@ const getExampleObject = ({
           value = value.find(v => v != null)
           arrays--
         }
-        return acc.concat(value)
+        if (_.isArray(value)) {
+          acc.push(...value)
```

Comment:
i need to test for Array? why not always add with `...`?

Reply:
Because if you spread an object it behaves very differently to just adding it